.circleci/config.yml

```diff
@@ -428,6 +429,15 @@ jobs:
       - store_artifacts:
           path: /tmp/detox_artifacts
 
+  pagopa_specs_diff:
+    docker:
+      - image: circleci/android:api-23-node8-alpha
```

Comment:
Why you are using the node image?

Reply:
@fpersico No specifics needs. I used that image for tests, because it is the same used in other jobs in this config. 
I changed it with **android:api-28** in the last commit. This job need only an environment where tare are these tools: **curl** and **diff**